sklearn/preprocessing/imputation.py

```diff
@@ -102,11 +102,25 @@ class Imputer(BaseEstimator, TransformerMixin):
         - If `axis=0` and X is encoded as a CSR matrix;
         - If `axis=1` and X is encoded as a CSC matrix.
 
+    add_missing_indicator : boolean, optional (default=False)
+        If True, the returned X will have an appended indicator matrix
+        to denote the features that have at least one missing value and
+        the indicator matrix has 1 for each missing value.
+        The shape of X will now be
+        ``(n_samples, n_features + n_features_with_missing)`` if axis=1
```

Comment:
Would something like ``(n_samples , n_features_new + len(imputed_features))`` be okay ?